src/ActionModal/components/ActionModal.vue

```diff
@@ -317,6 +317,10 @@ export default {
       type: Function,
       default: undefined
     },
+    validatorMoniker: {
```

Comment:
This makes the actionmodal more dependent on the wrapping application. I would recommend querying this from the actionmanager. What do you think @colw ?

Reply:
Shouldn't this info be in the `fromOptions`?

Reply:
The problem is we are treating the validator and the moniker as separate things.